lib/bracket-matcher-view.coffee

```diff
@@ -109,7 +125,10 @@ class BracketMatcherView
   findMatchingEndPair: (startPairPosition, startPair, endPair) ->
     return if startPair is endPair
 
-    scanRange = new Range(startPairPosition.traverse([0, 1]), startPairPosition.traverse([MAX_ROWS_TO_SCAN, 0]))
+    scanRange = new Range(
```

Comment:
I wonder if we could just mutate the point directly to avoid a bunch of point allocations.